ibis/bigquery/client.py

```diff
@@ -118,6 +108,15 @@ def dataset_id(self):
     def _table_expr_klass(self):
         return ir.TableExpr
 
+    def table(self, *args, **kwargs):
+        t = super(BigQueryClient, self).table(*args, **kwargs)
+        if '_PARTITIONTIME' in t.columns:
+            assert 'PARTITIONTIME' not in t
+            return (t
+                    .mutate(**{'PARTITIONTIME': t._PARTITIONTIME})
```

Comment:
Any reason not to do `.mutate(PARTITIONTIME=t._PARTITIONTIME)`?

Reply:
ideally, there's a variable for that magical string, in which case you'd need the `{}` form

Reply:
I added a variable for the magical string